### PR TITLE
Changes Default Values of Roles for Initial Page Load on Leaderboard Page 

### LINKED ISSUE #289: Change Default Values of Roles for Initial Page Load on Leaderboard Page
Currently, the initial page load displays data for all roles. However, to enhance the user experience, it is proposed to show data specifically for interns and contributors by default.

### PR DESCRIPTION
**Pull request by UdaySagar-Git**
fixes #289

### CODE REVIEW

**Comment by UdaySagar-Git** [comment]
@rithviknishad should i update home page leader board page with default roles or 
let it be for all roles

File: app/leaderboard/_components/LeaderboardWrapper.tsx [hunk 1]
```diff
@@ -16,7 +17,7 @@ export default async function LeaderboardWrapper({
       | "intern"
       | "operations"
       | "contributor"
-    )[]) ?? [],
+    )[]) ?? deaultRoles.map((i) => i.value),
```
Comment (rithviknishad) [inline app/leaderboard/_components/LeaderboardWrapper.tsx:20]: Let's keep this configurable as different organizations require different behaviour. Move this to the environment: `NEXT_PUBLIC_LEADERBOARD_DEFAULT_ROLES=comma,seperated,values`

File: lib/const.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const deaultRoles = [
+  { value: "intern", text: "Intern" },
+  { value: "contributor", text: "Contributor" },
+];
```
Comment (rithviknishad) [inline lib/const.ts:4]: Not required once moved to env

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Also update the home page

**Comment by UdaySagar-Git** [comment]
@rithviknishad can you please review

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**